app/services/message_status/main.py

```diff
@@ -0,0 +1,25 @@
+import json
+import logging
+from app.services.message_status import status_recorder, request_verifier
+from flask import jsonify
+
+def create_message_status(req_body, headers):
```

Comment:
The pattern in the Flask documentation is to implement the request routing in `app/routes`.

Reply:
Perhaps the most pragmatic thing to do is put these controller-like modules into a separate directory like `app/routes/` or `app/controllers/`. This gives us the ability to test them in the same way for now. 
I think ultimately we will want to refactor tests to call the the Flask specific parts of the routing ie controller integration tests. 
I don't think this code is service-like so at least moving it would help.